index.rst

```diff
@@ -8,7 +8,9 @@ Welcome to the ros2_control documentation!
 
    getting_started.rst
    ros2_controllers/doc/controllers_index.rst
+   ros2_control/ros2controlcli/doc/index.rst
```

Comment:
I would add index from ros2_control. Then we don't have to touch this anymore, but work in only one repo. E.g.:

```suggestion
   ros2_control/doc/index.rst
```

Reply:
happy to make that change once we have a  top level page on the other side, this as-is would break things when merged